src/services/Ssh.php

```diff
@@ -99,7 +99,7 @@ public function upload($src, $target)
      */
     public function download($src, $target)
     {
-        $cmd = "ssh {$this->remote} 'cat {$src} | gzip' | zcat > {$target}";
+        $cmd = "ssh {$this->remote} \"cat {$src} | gzip | zcat > {$target}\"";
```

Comment:
Did you change the wrapping intentionally? IMO it's a mistake.
Only the `cat {$src} | gzip` part should be executed on the remote.

Reply:
Oops, that was a mistake, thanks for catching it!

Reply:
Fixed.